services/src/main/java/org/keycloak/services/HttpResponseImpl.java

```diff
@@ -44,6 +56,31 @@ public void setHeader(String name, String value) {
         delegate.getOutputHeaders().putSingle(name, value);
     }
 
+    @Override
+    public void addCookieIfAbsent(HttpCookie cookie) {
```

Comment:
`addCookieIfAbsent` is a bit confusing name, as I'd say that means add the cookie if not already set in previous requests. Would it be okay to just call it `setCookie`? Or, guess if we want something more explicit `setCookieIfNotSet` or something like that?

Reply:
Changed to `setCookieIfNotSet`.